Fix #185

PatrickHaussmann opened the pull request:
RKI renamed the column in the Excel file

PatrickHaussmann committed:
Fix #185

RKI renamed the column in the Excel file

Rubber1Duck reviewed (changes requested):
RKI changed this very often! in the past they switched between 
"Punktschätzer des 4-Tage R-Wertes" and "Punktschätzer der 4-Tage R-Wert".
and i'm afraid they change back tomorrow and remove the "-".
So the better way should be

`let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert" || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];`

PatrickHaussmann committed:
rValue: test all the old excel column names too

In the case that the rki changes the heading in Excel to one of these versions again in the future

as suggested by Rubber1Duck: 
https://github.com/marlon360/rki-covid-api/pull/186#pullrequestreview-642070489

Rubber1Duck commented on src/data-requests/r-value.ts:
```diff
@@ -9,7 +9,7 @@ function parseRValue(data: ArrayBuffer): { r: number, date: Date } | null {
     const dateString = latestEntry["Datum des Erkrankungsbeginns"];
-    let rValue = latestEntry["Punktschätzer des 4-Tage-R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert"];
+    let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert" || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];
 
```
ups, there is a ] missing!
let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert"] || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];

PatrickHaussmann replied:
sorry, I overlooked that.

Rubber1Duck replied:
I suggested that also wrong! so thats my misstake!

PatrickHaussmann committed:
fix typo: missing bracket

Rubber1Duck reviewed (approved):
Now its working well!

marlon360 reviewed (approved):
Thanks @PatrickHaussmann
